src/gc.h

```diff
@@ -11,20 +11,24 @@
 
 #define V7_DISABLE_GC
 
-#define MARK(p) (* (uintptr_t *) &(p) |= 1)
+#define MARK(p) (* (unsigned long *) &(p) |= 1)
 
 /* call only on already marked values */
-#define UNMARK(p) (* (uintptr_t *) &(p) &= ~1)
+#define UNMARK(p) (* (unsigned long *) &(p) &= ~1)
 
-#define MARKED(p) ((uintptr_t) (p) & 1)
+#define MARKED(p) (((unsigned long) (void *) (p)) & 1)
 
 struct gc_tmp_frame {
   struct v7 *v7;
   size_t pos;
 };
 
+#ifdef _WIN32
+#define GC_TMP_FRAME(v) struct gc_tmp_frame v = new_tmp_frame(v7);
```

Comment:
You can take over - but I think the easier thing is to merge this one and
remove gcc extension in a separate PR

On Fri, Feb 27, 2015 at 3:39 PM, Marko Mikulicic <notifications@github.com>
wrote:

> In src/gc.h <https://github.com/cesanta/v7/pull/298#discussion_r25514186>:
>
> >
> >  struct gc_tmp_frame {
> >    struct v7 *v7;
> >    size_t pos;
> >  };
> >
> > +#ifdef _WIN32
> > +#define GC_TMP_FRAME(v) struct gc_tmp_frame v = new_tmp_frame(v7);
>
> Do you want me to take over this branch and resolve conflicts so I can
> make a pass at removing this GCC extension use?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cesanta/v7/pull/298/files#r25514186>.
>